src/components/Bookcontainer.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Booklist from './Booklist';
+import Bookinput from './Bookinput';
+import Category from './Category';
+
```

Comment:
To follow the best practice of `react` let's rename the components and name them to start with uppercase as you have and also the second ones to like rather than `Booklist` should be `BookList`, rather than `Bookinupt` should be `BookInput` and etc. After renaming the components' names don't forget to rename the names of your functions to keep consistency.